draft-ietf-quic-transport.md

```diff
@@ -2089,15 +2089,17 @@ different local addresses, as discussed in {{connection-id}}.  For this to be
 effective endpoints need to ensure that connections IDs they provide cannot be
 linked by any other entity.
 
-This eliminates the use of the connection ID for linking activity from
-the same connection on different networks.  Header protection ensures
-that packet numbers cannot be used to correlate activity.  This does not prevent
-other properties of packets, such as timing and size, from being used to
-correlate activity.
+An endpoint MUST use a new connection ID if it initiates connection migration,
+unless the peer has selected a zero-length connection ID.  This eliminates the
+use of the connection ID for linking activity from the same connection on
+different networks.  Header protection ensures that packet numbers cannot be
+used to correlate activity.  This does not prevent other properties of packets,
+such as timing and size, from being used to correlate activity.
 
 Clients MAY move to a new connection ID at any time based on
 implementation-specific concerns.  For example, after a period of network
-inactivity NAT rebinding might occur when the client begins sending data again.
+inactivity, where NAT rebinding might occur when the client begins sending data
```

Comment:
```suggestion
inactivity, NAT rebinding may cause an unintentional change in path when the client begins sending data
```

Reply:
Not adding; linking to #2441 where this separate issue will be discussed.